### ISSUE TITLE
SDK feed - time-stamp on moments enhancement

### ISSUE THREAD

**Issue by mawlle** [stagecast/bug-report#159]
See topic in this issue: https://github.com/stagecast/web-frontend/issues/430

**Summary**
If you want to specify a "sender name" on telegram or picture moments the space for the sender name-text is often taken by the time-stamp. A solution would be to make the time stamps smaller.
Therefore I suggest the following changes:

**Swedish**
"days": "dagar sen", -> "1d sen"
"hours": "timmar sen", -> "1t sen"
"minutes": "minuter sen", -> "2m sen"
"seconds": "sekunder sen" -> "1s sen"

**English**
"days": "days ago", -> "2d ago"
"hours": "hours ago", -> "2h ago"
"minutes": "minutes ago", -> "3m ago"
"seconds": "seconds ago" -> "5s ago"

***

**Comment by mawlle** [comment]
@Robin9224 and @peffis  do you agree to this?

If so and possible put it in.

I can commit to the repo, let me know what branch to commit to.

1. I can make the changes in these files (if you want and let me know what branch)
https://github.com/stagecast/stagecast-moments/blob/master/translations/en.json
https://github.com/stagecast/stagecast-moments/blob/master/translations/sv.json

2. In the current timestamps there is a whitespace between the number and the follwing text like this:
"1 sekund sen" if we want to do "1s sen" we should adjust that whitespace as well.

**Comment by mawlle** [comment]
**UPDATE**
Robin thinks is ok - then I say we go for it.

Stefan: Let me know if you want me to just go in there and change the text.

Also there is another text thing in the same file we need to change:
Change EN &  SWE to:

```
"sponsoredBy": {
"title": "This Moment is presented by:"
```

```
"sponsoredBy": {
"title": "Detta Moment presenteras av:"
```

**Comment by peffis** [comment]
Fixed (by markus). Will be included in 0.0.16
